provisioning/roles/gulp/templates/webpack.config.js

```diff
@@ -30,9 +30,9 @@ module.exports = function(production) {
         {
           loader: 'babel',
           exclude: /(node_modules|bower_components)/,
-          test: /\.jsx?$/,
+          test: /\.js$/,
           query: {
-            presets: ['es2016', 'react']
+            presets: ['es2015']
```

Comment:
I just rollback to leave transcoding jsx with react preset. 
Preset es2015 was missing as it's modulaire and incremental, es2016 don't add much ecmascript feature.